_overviews/scala3-migration/incompat-syntactic.md

```diff
@@ -38,135 +38,172 @@ It is composed of:
 - `=>>`
 - `?=>`
 
-For instance, the following piece of code can be compiled with Scala 2.13 but not wtih Scala 3.
+{% tabs scala-3-keywords_1 %}
+{% tab 'Scala 3 Only' %}
 
-```scala
-object given { // Error: given is now a keyword
-  val enum = ??? // Error: enum is now a keyword
+For instance, the following piece of code can be compiled with Scala 2.13 but not with Scala 3.
+~~~ scala
+object given { //Error: given is now a keyword
+  val enum = ??? //Error: enum is now a keyword
 
-  println(enum) // Error: enum is now a keyword
+  println(enum) //Error: enum is now a keyword
 }
-```
+~~~
+{% endtab %}
+{% endtabs %}
 
-The [Scala 3 migration compilation](tooling-migration-mode.html) rewrites the code into:
+{% tabs scala-3-keywords_2 %}
+{% tab 'Scala 3 Only' %}
 
-{% highlight diff %}
--object given {
-+object `given` {
--  val enum = ???
-+  val `enum` = ???
+The [Scala 3 migration compilation](tooling-migration-mode.html) rewrites the code into:
+~~~ scala
+object `given` {
+  val `enum` = ???
 
--  println(enum)
-+  println(`enum`)
+  println(`enum`)
 }
-{% endhighlight %}
```

Comment:
I think in this case the “diff” view is more relevant, because the goal is to show the changes applied by the rewriting tool.